.github/workflows/deploy.yml

```
@@ -0,0 +1,82 @@
+            
+            # 3. 기존에 실행 중인 컨테이너가 있으면 중지 및 제거 (없어도 오류 발생 안 함)
+            docker stop moodflix-container || true
+            docker rm moodflix-container || true
+            
+            # 4. 새 이미지로 컨테이너 실행
+            docker run -d -p 80:8080 --name moodflix-container \
+              -e JWT_SECRET=${{ secrets.JWT_SECRET }} \
+              -e KAKAO_CLIENT_ID=${{ secrets.KAKAO_CLIENT_ID }} \
+              -e TMDB_API_KEY=${{ secrets.TMDB_API_KEY }} \
+              -e DB_PASSWORD=${{ secrets.DB_PASSWORD }} \
+              ${{ steps.login-ecr.outputs.registry }}/${{ secrets.AWS_ECR_REPOSITORY }}:${{ github.sha }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)